bigchaindb/commands/bigchain.py

```diff
@@ -243,63 +238,55 @@ def run_load(args):
     workers.start()
 
 
+@configure_bigchaindb
 def run_set_shards(args):
```

Comment:
As far as I can tell, `run_set_shards()` and `run_set_replicas()` should have been autoconfiguring before (since they require a db connection, which may be affected by the config).

Unless this was intended to be this way (to not configure)? Maybe @sbellem or @r-marques you might be able to confirm.